chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -308,62 +313,41 @@
     end
   end
 else
-  if node["platform_version"].to_f < 12.0
-    cookbook_file "/etc/init.d/crowbar" do
-      owner "root"
-      group "root"
-      mode "0755"
-      action :create
-      source "crowbar.suse"
-    end
-
-    link "/usr/sbin/rccrowbar" do
-      action :create
-      to "/etc/init.d/crowbar"
-    end
-
-    # Make sure that any dependency change is taken into account
-    bash "insserv crowbar service" do
-      code "insserv crowbar"
-      action :nothing
-      subscribes :run, resources("cookbook_file[/etc/init.d/crowbar]"), :delayed
-    end
-  else
-    cookbook_file "/etc/tmpfiles.d/crowbar.conf" do
-      owner "root"
-      group "root"
-      mode "0644"
-      action :create
-      source "crowbar.tmpfiles"
-    end
+  cookbook_file "/etc/tmpfiles.d/crowbar.conf" do
+    owner "root"
+    group "root"
+    mode "0644"
+    action :create
+    source "crowbar.tmpfiles"
+  end
 
-    bash "create tmpfiles.d files for crowbar" do
-      code "systemd-tmpfiles --create /etc/tmpfiles.d/crowbar.conf"
-      action :nothing
-      subscribes :run, resources("cookbook_file[/etc/tmpfiles.d/crowbar.conf]"), :immediately
-    end
+  bash "create tmpfiles.d files for crowbar" do
+    code "systemd-tmpfiles --create /etc/tmpfiles.d/crowbar.conf"
+    action :nothing
+    subscribes :run, resources("cookbook_file[/etc/tmpfiles.d/crowbar.conf]"), :immediately
+  end
 
-    # Use a systemd .service file on SLE12
-    cookbook_file "/etc/systemd/system/crowbar.service" do
-      owner "root"
-      group "root"
-      mode "0644"
-      action :create
-      source "crowbar.service"
-    end
+  # Use a systemd .service file on SLE12
+  cookbook_file "/etc/systemd/system/crowbar.service" do
+    owner "root"
+    group "root"
+    mode "0644"
+    action :create
+    source "crowbar.service"
+  end
 
-    # Make sure that any dependency change is taken into account
-    bash "reload systemd after crowbar update" do
-      code "systemctl daemon-reload"
-      action :nothing
-      subscribes :run, resources("cookbook_file[/etc/systemd/system/crowbar.service]"), :immediately
-    end
+  # Make sure that any dependency change is taken into account
+  bash "reload systemd after crowbar update" do
+    code "systemctl daemon-reload"
+    action :nothing
+    subscribes :run, resources("cookbook_file[/etc/systemd/system/crowbar.service]"), :immediately
+  end
 
-    link "/usr/sbin/rccrowbar" do
-      action :create
-      to "service"
-    end
+  link "/usr/sbin/rccrowbar" do
+    action :create
+    to "service"
   end
+end
```

Comment:
See https://github.com/crowbar/crowbar-core/pull/45/files?w=1#diff-f95f7579134d98ee1a8cc86d5e43f236L311

I see the `if` and the `else` being dropped, but no `end` being dropped.